pyiron/lammps/base.py

```diff
@@ -920,6 +968,13 @@ class Input:
     def __init__(self):
         self.control = LammpsControl()
         self.potential = LammpsPotential()
+        self.bond_dict = dict()
```

Comment:
Should not the bond dict be empty by default?

Reply:
I thought about this a little bit. I think it is important to have some pre-defined default values for `bond_dict` just as an example. Once again, these bonds will only be drawn if water molecules are present (not just O or H atoms). The users can modify/play with these parameters if they want to. I don't think it is a good idea to force the user to define these bond parameters every time water is present since a certain bit of convenience is necessary.

Reply:
We can have an example as comment, or have a separate function the user has to call, but we should not have any bonds by default, because then the behavior for water is different compared to other molecules.

Reply:
But that would not be ideal. If the bonds are not drawn by default, the system would simply run the calculation without any error messages (since technically it is still a valid potential) which is highly misleading. This is dangerous since many people already use the water example on our website. So for the sake of backward compatibility, I think we should allow just these default bond parameters at least for water (most of the simulations using this style use water anyway). I would rather that users who do not want water bonds to be drawn (who are very rare) to use the `define_bonds()` function explicitly. Besides `define_bonds()` has very LAMMPS specific parameters (eg. bond/angle type) which I think only users who know what they are doing should modify.

Reply:
It all comes down to the zen of python `Special cases aren't special enough to break the rules.` You could raise a warning that no bonds were defined.

Reply:
I disagree. I don't think this violates the zen of python.